GliaWidgets/Sources/RemoteConfiguration/RemoteConfiguration+CallVisualizer.swift

```diff
@@ -24,6 +24,6 @@ extension RemoteConfiguration {
     struct NumberSlot: Codable {
         let background: Layer?
         let font: Font?
-        let fontColor: Color?
+        let foreground: Color?
```

Comment:
I raised this question in schema repo, however, does it look like `Text`?

Reply:
Almost. I thought to use `Text`, but `Text` contains `background` as `Color` and `NumberSlot` contains `background` as `Layer`

Reply:
5ba9dd172544bb117127667abf2b04729e058ca8